docs/konnectors.md

```diff
@@ -320,25 +320,56 @@ On the host create the file /etc/rkt/net.d/10-containers.conf
 
 and run your container with the "--net=bridge" option. That way, a new interface is available in the container and gives you access to the host.
 
+## Konnector install and run details
+
+### Install
+
+The konnectors will be installed in the .cozy_konnectors directory which is in the VFS using git clone
+(like the apps at the moment).
+
+The konnectors installation may be triggered when the user says he wants to use it. The resulting
+repository is then kept for each run of the konnector. It may then be given to the user the
+possibility to upgrade the konnector to the latest version if any.
+
+To update a given konnector, a ```git pull``` command is run on the konnector.
+
+### Details about running a konnector
+
+To run a given konnector, the stack will copy this connector in a "run" directory, which is not in
+the VFS. This directory will be given to the rocket container as the current working directory
+with full read and write access on it. This is where the container will put its logs and any temp
+file needed. There will be also cozy-client.js and the shared libraries in a lib directory inside
```

Comment:
The shared libraries are the tools, specific to the konnectors to help developer to write its connector. It corresponds to the actual content of the [lib](https://github.com/cozy-labs/konnectors/tree/master/server/lib) directory. Anyway, it is mandatory to allow easy migration of existing konnectors at least.

For the second part, yes, the konnectors will have to contain their own dependencies (npm packages, etc) in their git repository.